extract/extract/utils.py

```diff
@@ -7,6 +8,11 @@
 SUSTAINABILITY_LABELS = load_and_get_sustainability_labels()
 logger = getLogger(__name__)
 
+_certificate_category_names = {
```

Comment:
I also thought about doing it like that, but I went for the current option, as it involves less manual work.
In addition, it would be great for future certificates to not add other label categories (synonymys) like `SMARTPHONE` or `MOBILE_PHONE` than we have currently. Doing so, requires no code changes and the certificate names will be more standardized.